src/fu-engine.c

```diff
@@ -2909,7 +2930,11 @@ fu_engine_load_plugins (FuEngine *self, GError **error)
 		if (name == NULL)
 			continue;
 		if (fu_engine_is_plugin_name_blacklisted (self, name)) {
-			g_debug ("%s is blacklisted", name);
+			g_debug ("plugin %s is blacklisted", name);
+			continue;
+		}
+		if (!fu_engine_is_plugin_name_whitelisted (self, name)) {
+			g_debug ("plugin %s is not whitelisted", name);
```

Comment:
Do you want whitelist to always override blacklist?  Right now I think if you put a plugin in both blacklist and whitelist the effect will be blacklist.

Reply:
I don't know. I'd say keep it simple for now, and then we can work out what needs to take precedence when we have a use case.